balarco/utils.py

```diff
@@ -8,6 +8,19 @@
 from rest_framework.authtoken.models import Token
 
 
+STATUS = (
+    (0, 'Pendiente'),
+    (1, 'Diseño'),
+    (2, 'Cuentas'),
+    (3, 'Validación'),
+    (4, 'Producción'),
+    (5, 'Por cobrar'),
+    (6, 'Por facturar'),
+    (7, 'Pagado'),
+    (8, 'Terminado')
```

Comment:
How often are we going to access this STATUS in other modules?? if it only going to be used in the models, leave it in the models. If it going to be used in views leave it here but turn it into a dictionary.